groclient/client.py

```diff
@@ -850,7 +850,10 @@ def get_top(self, entity_type, num_results=5, **selection):
             self.access_token, self.api_host, entity_type, num_results, **selection
         )
 
-    def get_df(self, show_revisions=False, index_by_series=False):
+    def get_df(self,
+               show_revisions=False,
+               include_names=False,
```

Comment:
It would be a bit safer to add `include_names` as the last argument. Adding it here could break existing code.

For example, the meaning of `client.get_df(True, True)` would change from setting index_by_series=True to setting index_by_series=False and include_names=True.

Reply:
done